src/quote/indexQuoteProvider.ts

```diff
@@ -70,13 +71,14 @@ export class FlashMintQuoteProvider
 {
   constructor(
     private readonly provider: JsonRpcProvider,
+    private readonly swapQuoteProvider?: SwapQuoteProvider,
```

Comment:
Right now this is only used for FlashMintZeroEx but once all leverage quote providers are using this as well, we could remove the `ZeroExApi` dependency below. This would make it more modular as the user of the SDK can decide what type of SwapQouteProvider to use.